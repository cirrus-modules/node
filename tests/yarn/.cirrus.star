load("../../lib.star", "node_script_task")

def main(ctx):
    return [node_script_task()]
