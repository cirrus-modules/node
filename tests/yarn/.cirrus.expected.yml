task:
  name: test
  container:
    image: node:latest
    cpu: 2
    memory: 4096
  node_modules_cache:
    folder: node_modules
    fingerprint_script:
      - cat yarn.lock
    populate_script:
      - yarn install
  test_script:
    - yarn run test
